src/lib/draw/Toolbox.svelte

```diff
@@ -44,15 +59,15 @@ repeatedly load anything. Make sure this is only created once, then just hidden.
   <div class="toolbar">
     <ToolButton setMode={{ mode: "new-point" }}>
       <img src={pointIcon} alt="New point" />
-      New point
+      New point (1)
```

Comment:
idk how I feel about. I don't think the average user will interpret this as 'this number represents a hotkey'

Reply:
I tried underlining (used in some dropdown menus to indicate hotkeys), but it looked odd. We could also hide the info in the help for each tool, but that's _really_ undiscoverable. Hmm..

Reply:
PTAL -- stopped showing the hotkey upfront, moved to help